web-ng/src/app/services/feature/feature.service.ts

```diff
@@ -93,8 +103,14 @@ export class FeatureService {
       layerId,
       new firestore.GeoPoint(lat, lng)
     );
-    return this.dataStore
-      .updateFeature(project.id, newFeature)
-      .then(() => this.selectFeature(newFeature.id));
+    await this.dataStore.updateFeature(project.id, newFeature);
+    return this.selectFeature(newFeature.id);
```

Comment:
`selectFeature` doesn't return a value, so this return isn't necessary.

Reply:
Oops these comes from IDE suggestion, I don't quite understand the best practices, so let me revert it for now.